From: Carsten Schoenert <c.schoenert@t-online.de>
Date: Mon, 21 Mar 2022 19:49:36 +0100
Subject: docs: Use the default theme instead of furo

Forwarded: Not-Needed
---
 docs/conf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/conf.py b/docs/conf.py
index 3556f81..e471b9b 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -51,7 +51,7 @@ exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = "furo"
+html_theme = "default"
 html_title = "Lunr.py"
 
 # Add any paths that contain custom static files (such as style sheets) here,
